lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -249,6 +233,19 @@ sub _resolve_reference {
 sub __errors__ {
     my $self   = shift;
     my @errors = $self->SUPER::__errors__;
+
+    for my $model (qw(wgs_model exome_model)) {
+        if ($self->$model) {
+            unless ($self->$model->isa('Genome::Model::SomaticVariation') || $self->$model->isa('Genome::Model::SomaticValidation')) {
```

Comment:
Seems like there ought to be a way to say "Things that have builds that implement the `SomaticInterface`" rather than calling these two out specifically.